plugins/chat/app/controllers/chat/api/current_user_threads_controller.rb

```diff
@@ -21,4 +21,11 @@ def index
       on_model_not_found(:threads) { render json: success_json.merge(threads: []) }
     end
   end
+
+  def thread_count
+    with_service(::Chat::LookupUserThreads) do
+      on_success { render json: success_json.merge(thread_count: result.threads.size) }
+      on_model_not_found(:threads) { render json: success_json.merge(thread_count: 0) }
+    end
```

Comment:
We are missing a spec for this, just test success and model not found in a controller spec